libshaderc/include/shaderc.h

```diff
@@ -63,6 +63,15 @@ typedef enum {
   shaderc_profile_es,
 } shaderc_profile;
 
+// Used in the result module (shaderc_spv_module) to indicate the status of an
```

Comment:
Strict stylists would say you should only describe the inherent meaning of the thing you're defining.  The definition should stand on its own.
Don't describe it by saying how it's used, since that will change over time. It gets out of date.
Ok for now.